benchmark.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+JAVA_ARGS=()
+JMH_ARGS=()
+BINARY="java"
+BENCHMARK_PATTERN=
+
+while getopts ":j:c:b:e:p:a:m:h" opt; do
+  case $opt in
+    j) JAVA_ARGS+=("${OPTARG}")
+    ;;
+    c) CLASSPATH_ARG="${OPTARG}"
+    ;;
+    b) BINARY="${OPTARG}"
+    ;;
+    p) PROFILER_ARG="${OPTARG:+-prof ${OPTARG}}"
```

Comment:
This is the core motivation for the change. If `-p` is specified with an non empty value this will set `PROFILER_ARG` to `-prof ${OPTARG}`. Without this it would end up adding `-prof " "` and thus JMH fails to initialise.